### PR TITLE
About me

### PR DESCRIPTION
**Pull request by mgirar84**

### CODE REVIEW

File: css_stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
+body {
+	font-family: "Arial", sans-serif;
+	margin: 5px;
+	padding: 5px;
+	margin-left: auto;
+	margin-right: auto;
```
Comment (ashleycoles) [inline css_stylesheet.css]: Try not to use margin and margin-left/right. Combine into a single margin rule if you can

File: css_stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	width: 1200px;
+}
+
+
```
Comment (ashleycoles) [inline css_stylesheet.css]: Remove extra line of whitespace

File: css_stylesheet.css [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+	font-family: "Courier New", serif;
+	color: #343A40;
+}
+
+.number {
+	font-weight: 900;
+	text-decoration: none;
+	font-family: "Courier New", serif;
+	color: #343A40;
+}
```
Comment (ashleycoles) [inline css_stylesheet.css]: This looks like it contains the same css as the .email class, could they be combined into a single class? Keep the code DRY.

File: css_stylesheet.css [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+.number:hover {
+	text-decoration-line: underline;
+	/*text-shadow: 1px 1px;*/
+	font-family: "Courier New", serif;
+	color: #343A40;
+}
```
Comment (ashleycoles) [inline css_stylesheet.css]: Could also combine with the .email:hover selector

File: css_stylesheet.css [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+.socialMediaButtons {
+	font-size: 25px;
+	margin-left: 39px;
+	margin-bottom: 15px;
+	color: #343A40;
+}
+
+
```
Comment (ashleycoles) [inline css_stylesheet.css]: Remove extra whitespace

File: css_stylesheet.css [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+.myName {
+	font-family: "Courier New", serif;
+	color: #343A40;
+	float: right;
+	margin-right: 39px;
+	margin-top: 55px;
+}
+
+/* #16AABF -blue
```
Comment (ashleycoles) [inline css_stylesheet.css]: Remove thing

File: html_reference.html [hunk 7]
```diff
@@ -2,10 +2,51 @@
 <html>
 	<head>
 		<link rel="stylesheet" href="css_stylesheet.css">
+		<script src="https://kit.fontawesome.com/c90803ff49.js" crossorigin="anonymous"></script>
 		<title>milo_girardet</title>
 	</head>
-	
+	<header class="contactDetails">
```
Comment (ashleycoles) [inline html_reference.html:9]: Seems to be missing a <body> tag around your content

File: html_reference.html [hunk 8]
```diff
@@ -2,10 +2,51 @@
...
+		<div class="buttonsSentLeft">
+				<ul class="contactAnchors">
+					<li>
+						<a href="#" class=email>milogirardet@gmail.com</a>
+					</li>
+					<li>
+						<a href="#" class=number>07519645711</a>
+					</li>
+				</ul>
```
Comment (ashleycoles) [inline html_reference.html]: Watch indents here

**Review by ashleycoles — COMMENTED** [review]
Few changes to make, but mostly good!

File: css_stylesheet.css (hunk 6 above)
Comment (mgirar84) [reply css_stylesheet.css]: saving colours for later

**Review by ashleycoles — APPROVED** [review]

**Pull request merged**